test/ex_doc/formatter/html_test.exs

```diff
@@ -537,4 +537,21 @@ defmodule ExDoc.Formatter.HTMLTest do
       end
     end
   end
+
+  describe "does not link to local modules with @moduledoc false" do
```

Comment:
Can we write unit tests instead through autolink_test instead of adding more modules to our integration suite? 🤔

Reply:
I'd rather not, because we will be modelling `:modules_refs` in config. I don't think a test like that would be of help

Reply:
Sorry, I don't understand it. it is probably my bad but can you please explain whta you mean we cannot unit test it? I understand at least ModuleDocFalse is necessary but I don't understanad why we need the other fixture.